plasmapy/particles/ionization_state.py

```diff
@@ -374,15 +531,18 @@ def _is_normalized(self, tol: Optional[Real] = None) -> bool:
     def normalize(self) -> None:
         """
         Normalize the ionization state distribution (if set) so that the
-        sum becomes equal to one.
+        sum of the ionic fractions becomes equal to one.
+
+        This method may be used, for example, to correct for rounding
+        errors.
         """
         self._ionic_fractions = self._ionic_fractions / np.sum(self._ionic_fractions)
 
     @property
     def equil_ionic_fractions(self, T_e: u.K = None):
```

Comment:
I'd argue `equil_ionic_fractions` and `equilibrate` should be removed altogether since they do nothing and would be misleading to users.  I can see the use in the future, but, since the functionality does not exist, there shouldn't be placeholders and an issue should be made to create the functionality.

Reply:
Yeah...I think you're right.